### PR TITLE
AGW: ubuntu: deploy: create missing interface config.

### PR DESCRIPTION
**Pull request by pshelar**
Different ubuntu flavors has different level of
in-built interface configuration.  Following change
assume no configuration present and builds entire
interface config file from scratch.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Test Plan
Validate it in lab with Ubuntu server edition.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ardzoht — APPROVED** [review]
lgtm

File: lte/gateway/deploy/agw_install_ubuntu.sh [hunk 1]
```diff
@@ -72,13 +73,16 @@ if [[ $1 != "$CLOUD_INSTALL" ]] && ( [[ ! $INTERFACES == *'eth0'*  ]] || [[ ! $I
 
   # interface config
   apt install -y ifupdown net-tools
+  mkdir -p "$INTERFACE_DIR"
+  echo "source-directory $INTERFACE_DIR" > /etc/network/interfaces
+
   echo "auto eth0
-  iface eth0 inet dhcp" > /etc/network/interfaces.d/eth0
+  iface eth0 inet dhcp" > "$INTERFACE_DIR"/eth0
```
Comment (ardzoht) [inline lte/gateway/deploy/agw_install_ubuntu.sh:80]: INTERFACE_DIR already contains trailing `/`? would this not result in double backlash?

**Review by ardzoht — APPROVED** [review]
minor comment

File: lte/gateway/deploy/agw_install_ubuntu.sh (hunk 1 above)
Comment (pshelar) [reply lte/gateway/deploy/agw_install_ubuntu.sh:80]: removed it.

**Pull request merged**